packages/synapse-react-client/src/synapse-client/HttpClient.ts

```diff
@@ -26,7 +26,7 @@ export const doPost = <T>(
     headers: {
       Accept: '*/*',
       'Access-Control-Request-Headers': 'authorization',
-      'Content-Type': 'application/json; charset=utf8',
+      'Content-Type': 'application/json; charset=UTF-8',
```

Comment:
Note that this is case insensitive.  See https://www.rfc-editor.org/rfc/rfc2046#section-4.1.2

> Unlike some other parameter values, the values of the charset parameter are NOT case sensitive.